Sources/TuistKit/Commands/MCPCommand.swift

```diff
@@ -0,0 +1,21 @@
+import ArgumentParser
+
+public struct MCPCommand: AsyncParsableCommand, TrackableParsableCommand {
+    public var analyticsRequired: Bool { false }
+
+    public init() {}
+
+    public static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "mcp",
+            abstract: "Start an MCP server to interface LLMs with your local dev environment.",
+            subcommands: [
+                MCPSetupCommand.self,
+            ]
+        )
+    }
+
+    public func run() async throws {
+        try await MCPCommandService().run()
+    }
```

Comment:
I'd consider keeping the `mcp` command for grouping only and run the actual command in `tuist mcp start`. I'd even consider hiding `tuist mcp start` from the `--help` as I wouldn't expect developers to run this directly.